core/include/moveit/task_constructor/task.h

```diff
@@ -117,11 +137,11 @@ class Task : protected WrapperBase
 	void init();
 
 	/// reset, init scene (if not yet done), and init all stages, then start planning
-	bool plan(size_t max_solutions = 0);
+	moveit::core::MoveItErrorCode plan(size_t error_code = 0);
```

Comment:
The argument name `max_solutions` should stay:
```suggestion
	moveit::core::MoveItErrorCode plan(size_t max_solutions = 0);
```